system_setup/cmd/server.py

```diff
@@ -74,6 +79,21 @@ def main():
     logger.info("Starting System Setup server revision {}".format(version))
     logger.info("Arguments passed: {}".format(sys.argv))
 
+    prefillFile = opts.prefill
+    if prefillFile:
+        if os.path.exists(prefillFile):
+            statInfo = os.stat(prefillFile)
+            mode = statInfo.st_mode
+            isRegularFile = (stat.S_ISREG(mode) != 0)
+            if not isRegularFile:
```

Comment:
https://docs.python.org/3/library/os.path.html#os.path.isfile maybe?

Reply:
Yeah, it also works fine. Thanks for the advice.